src/legacy/core_plugins/vega/public/data_model/__tests__/vega_parser.js

```diff
@@ -116,8 +116,8 @@ describe('VegaParser._parseSchema', () => {
     expect(vp.spec).to.eql({ $schema: 'https://vega.github.io/schema/vega/v3.0.json' });
```

Comment:
I wonder if we should switch the default specification to Vega 4 now. But that might break existing Vega visualisations, so I think we can't do it :thinking:  Meaning from now on the default spec will always stay Vega 3.0. Kind of stupid, but maybe we should have forbidden specs without a schema in the first place.

Reply:
Good catch - Suggestion: We remove the default version as a breaking change in 8.0 and leave it at 3.0 for now

Reply:
I like that suggestion. I'll open an issue, for that plan.

Reply:
https://github.com/elastic/kibana/issues/30951